cli/internal/cmd/iamcreateazure.go

```diff
@@ -26,6 +28,8 @@ func newIAMCreateAzureCmd() *cobra.Command {
 		RunE:  runIAMCreateAzure,
 	}
 
+	cmd.Flags().String("subscriptionID", "", "subscription ID of the Azure account (required)")
+	must(cobra.MarkFlagRequired(cmd.Flags(), "subscriptionID"))
```

Comment:
shouldn't be marked because one can also use ARM_SUBSCRIPTION_ID, no?

Reply:
If the flag is marked required, users need to pass `--subscriptionID ''` if they want to use the env var. That doesn't seem what you intended?

> Alternatively, we could also not mark this flag required, but then check if the env variable is set when parsing the flag

Aren't you already doing this?

Reply:
You're right :laughing: 
I'll remove the required annotation